lib/rex/post/meterpreter/packet.rb

```diff
@@ -397,9 +397,14 @@ def inspect
     end
     group ||= (self.class.to_s =~ /Packet/)
     if group
+      has_command_ids = ((self.method == COMMAND_ID_CORE_ENUMEXTCMD || self.method == COMMAND_ID_CORE_LOADLIB) && type == PACKET_TYPE_RESPONSE)
       tlvs_inspect = "tlvs=[\n"
       @tlvs.each { |t|
-        tlvs_inspect << "  #{t.inspect}\n"
+        if t.type == TLV_TYPE_UINT && has_command_ids
+          tlvs_inspect << "  #{t.inspect[0..-2]} command=#{::Rex::Post::Meterpreter::CommandMapper.get_command_name(t.value)}>\n"
```

Comment:
Is the `[0..-2]` to get rid of `\n` or similar? 👀 


```suggestion
          tlvs_inspect << "  #{t.inspect.chomp} command=#{::Rex::Post::Meterpreter::CommandMapper.get_command_name(t.value)}>\n"
```

https://apidock.com/ruby/String/chomp

Reply:
Feature request: It may be nice to ljust the value, so it would look like this before:

https://apidock.com/ruby/v2_5_5/String/ljust

```
  #<Rex::Post::Meterpreter::Tlv type=COMMAND-ID      meta=INT        value=10 command=core_enumextcmd>
  #<Rex::Post::Meterpreter::Tlv type=UUID            meta=RAW        value="\xFC!\xF0[\xD5\x88\x0Fu\x87\xE7\x92\xF3\xE5\xF1\x ...">
  #<Rex::Post::Meterpreter::Tlv type=UINT            meta=INT        value=17 command=core_patch_url>
  #<Rex::Post::Meterpreter::Tlv type=UINT            meta=INT        value=8 command=core_channel_write>
```

And after:
```
  #<Rex::Post::Meterpreter::Tlv type=COMMAND-ID      meta=INT        value=10 command=core_enumextcmd>
  #<Rex::Post::Meterpreter::Tlv type=UUID            meta=RAW        value="\xFC!\xF0[\xD5\x88\x0Fu\x87\xE7\x92\xF3\xE5\xF1\x ...">
  #<Rex::Post::Meterpreter::Tlv type=UINT            meta=INT        value=17 command=core_patch_url>
  #<Rex::Post::Meterpreter::Tlv type=UINT            meta=INT        value=8  command=core_channel_write>
```